cli/cmd/lql_show.go

```diff
@@ -0,0 +1,69 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// lqlShowCmd represents the lql show command
+	lqlShowCmd = &cobra.Command{
+		Use:   "show <query_id>",
+		Short: "show an LQL query",
+		Long:  `Show an LQL query.`,
+		Args:  cobra.MaximumNArgs(1),
```

Comment:
Lets change all other commands that have a required argument to follow this pattern:
```suggestion
		Args:  cobra.ExactArgs(1),
```

Reply:
Fixed.